app/util/planParamUtil.js

```diff
@@ -190,6 +190,13 @@ const getShouldMakeCarQuery = (
   );
 };
 
+const getShouldMakeOnDemandTaxiQuery = time => {
+  const date = new Date(time * 1000);
+  return (
+    date.getHours() > 22 || (date.getHours() === 22 && date.getMinutes() > 0)
```

Comment:
@leonardehrenfried What is the desired end-time for executing on demand taxi queries?

Reply:
5 AM.

And also please start it at 9PM.

Reply:
Ok. But if I execute queries with departure time before 22:01 api doesn't return any results.

Reply:
Hmm, maybe I'm wrong then. I will ask the experts. @derhuerst @hbruch

Reply:
There isn't a single time where the on-demand traffic starts, it varies by line and day.

For example, on Sundays, the [779 line](https://stadtwerke.herrenberg.de/fileadmin/user_upload/www/OEPNVUndParken/OEPNV/Fahrplaene/Fahrplaene_2021/Linie_779.pdf) is on-demand from 19:51, but on weekdays from 21:05; The 780 line on the other hand has fixed (*not* on-demand) runs until 10:30.

Flex routing still uses fixed (*not* on-demand) runs, and this is only a matter of performance, right? To be on the safe side, we could use 9pm and 5am, as proposed by Leonard.